src/tests/Interop/COM/NativeClients/windows_version_helpers.h

```diff
@@ -33,27 +33,27 @@ inline HRESULT is_windows_nano()
 
```

Comment:
I don't think `GetProductInfo` (above) sets the last error, so we shouldn't use `GetLastError` for the return value on failure.